apps/admin_app/lib/admin_app_web/templates/role/_form.html.eex

```diff
@@ -1,8 +1,11 @@
 <%= form_for @changeset, @action, [as: :role], fn f -> %>
   <%= input f, :name %>
   <%= input f, :description %>
+  
+  <%= label f, :permissions, "Permissions" %>
   <%= multiple_select(f, :permissions, formatted_list(:permissions),
-        class: "form-control custom-select")%>
+        prompt: "Add Permission", class: "full-width w-100", "data-init-plugin": "select2")%>
```

Comment:
Multiselect input helper missing

Reply:
I will add an issue for this, I will pick this up later on.